environment/src/test/java/com/sequenceiq/environment/environment/flow/deletion/EnvDeleteActionsTest.java

```diff
@@ -178,52 +178,36 @@ void setUp() {
     @Test
     void idbmmsDeleteActionTestHappyPath() {
         testDeleteActionHappyPath(underTest::idbmmsDeleteAction, DELETE_IDBROKER_MAPPINGS_EVENT.selector(),
-                EnvironmentStatus.IDBROKER_MAPPINGS_DELETE_IN_PROGRESS, ResourceEvent.ENVIRONMENT_IDBROKER_MAPPINGS_DELETION_STARTED);
-    }
-
-    @Test
```

Comment:
Why is the deletion?

Reply:
Those test cases were invalid, they were supposed to verify a scenario that does not apply (any more) to these delete actions. In particular, there is no distinction regarding the "environment missing" case in the action logic. (This is in direct opposition with the create actions that do differentiate between environment absent & present.)

Reply:
ok